spec/spec_helper.rb

```diff
@@ -90,7 +90,7 @@
   # order dependency and want to debug it, you can fix the order by providing
   # the seed, which is printed after each run.
   #     --seed 1234
-  config.order = :random
+  config.order = :random unless config.files_to_run.one?
```

Comment:
What is this for?

Reply:
so the specs run in order when I'm working with a single spec file ... it's easier to find the patterns of what is working and what isn't (e.g. if all the new ones are on the bottom).